SpatialGDK/Source/SpatialGDKFunctionalTests/SpatialGDK/RegisterAutoDestroyActorsTest/RegisterAutoDestroyActorsTest.cpp

```diff
@@ -50,44 +50,40 @@ void ARegisterAutoDestroyActorsTestPart1::PrepareTest()
 					++NumCharactersFound;
 				}
 
-				if (NumCharactersFound == NumCharactersExpected)
-				{
-					FinishStep();
-				}
+				RequireEqual_Int(NumCharactersFound, NumCharactersExpected,
+								 TEXT("Clients should observe the correct number of characters in their world."));
+				FinishStep();
 			},
 			5.0f);
 	}
 
 	{ // Step 3 - Destroy by all servers that have authority
 		AddStep(
-			TEXT("SERVER_ALL_RegisterAutoDestroyActors"), FWorkerDefinition::AllServers,
-			[this]() -> bool {
+			TEXT("SERVER_ALL_RegisterAutoDestroyActors"), FWorkerDefinition::AllServers, nullptr, nullptr,
+			[this](float DeltaTime) {
 				int NumCharactersFound = 0;
-				int NumCharactersExpected = 1;
+				int NumCharactersExpected = 1; // 1, because we filter by authority
```

Comment:
minor: `uint32`?

Reply:
hmm, I don't know if we actually have a preference here, we seem to use just `int` in quite a few places